dojo_plugin/api/__init__.py

```diff
@@ -14,9 +16,35 @@
 from .v1.workspace import workspace_namespace
 from .v1.search import search_namespace
 from .v1.test_error import test_error_namespace
+from .v1.user import user_namespace
 
 api = Blueprint("pwncollege_api", __name__)
 
+# Get CORS origin from environment variable
+cors_origin = os.environ.get('CORS_ORIGINS')
```

Comment:
Let's add this to `dojo_plugin/config.py` like our other configuration values, and import it into here from there.